test/topology_custom/test_tablets2.py

```diff
@@ -1718,3 +1718,39 @@ async def test_decommission_not_enough_racks(manager: ManagerClient):
         expected_replicas_per_server = get_expected_replicas_per_server(live_servers.values(), dead_servers.values(), ctx.initial_tablets, ctx.rf)
         tablet_count = await get_tablet_count_per_shard_for_hosts(manager, all_servers.values(), tables)
         verify_replicas_per_server("After decommission", expected_replicas_per_server, tablet_count, ctx.initial_tablets, ctx.rf)
+        
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_truncate_during_topology_change(manager: ManagerClient):
+    """Test truncate operation during topology change."""
+
+    # Start 3 node cluster
+    cfg = {'enable_tablets': True}
+    servers = await manager.servers_add(3, config=cfg)
+    cql = manager.get_cql()
+    await cql.run_async("CREATE KEYSPACE ks WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 3}")
+    await cql.run_async("CREATE TABLE ks.test (k int PRIMARY KEY, v int)")
+
+    logger.info("Populating table")
+    keys = range(10000)
+    stmt = cql.prepare("INSERT INTO ks.test (k, v) VALUES (?, ?)")
+    stmt.consistency_level = ConsistencyLevel.ONE
```

Comment:
Why do you have 3 servers but CL ONE?

Reply:
Used ONE for quick testing, will change it to QUORUM.